gui-linux sample compilation error in intel mac

**Describe the bug**
GUI linux sample is not compiling in intel mac machine

**To Reproduce**
Steps to reproduce the behavior:
1. Run gui-linux in intel mac

**Expected behavior**
Should compile properly

**Log**
```
# github.com/Code-Hex/vz/example/gui-linux
./main.go:416:39: undefined: createRosettaDirectoryShareConfiguration
make: *** [build] Error 2
```

**Environment that you use to compile (please complete the following information):**

 - Xcode version: [`Xcode 14.1
Build version 14B47b
`]
 - macOS Version: [`ProductName:		macOS
ProductVersion:		13.0
BuildVersion:		22A380
`]
 - mac architecture: [`x86_64
`]
 - Go Version: [`go version go1.19.2 darwin/amd64
`]

--- a/.github/workflows/compile.yml
+++ b/.github/workflows/compile.yml
@@ -48,7 +48,5 @@ jobs:
         run: make test
       - name: Build Linux
         run: make -C example/linux
-      - name: Build macOS
-        run: make -C example/macOS
       - name: Build GUI Linux
         run: make -C example/gui-linux

--- a/.github/workflows/compile.yml
+++ b/.github/workflows/compile.yml
@@ -40,11 +40,15 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
           cache: true
+      - name: vet
+        run: go vet ./...
       - name: Download Linux kernel
         run: make download_kernel
       - name: Unit Test
         run: make test
-      - name: Build
-        run: cd example/linux && make
-      - name: vet
-        run: go vet ./...
+      - name: Build Linux
+        run: make -C example/linux
+      - name: Build macOS
+        run: make -C example/macOS
+      - name: Build GUI Linux
+        run: make -C example/gui-linux

--- a/example/gui-linux/rosseta_directory_share.go
+++ b/example/gui-linux/rosseta_directory_share.go
@@ -1,5 +1,5 @@
-//go:build !darwin && !arm64
-// +build !darwin,!arm64
+//go:build darwin && !arm64
+// +build darwin,!arm64
 
 package main
 

